pkg/controllers/clusterset/syncclusterrolebinding/syncclusterrolebinding_controller.go

```diff
@@ -50,10 +48,6 @@ func NewReconciler(kubeClient kubernetes.Interface,
 
 // Run a routine to sync the clusterrolebinding periodically.
 func (r *Reconciler) Run(ctx context.Context, period time.Duration) {
-	if ok := kubecache.WaitForCacheSync(ctx.Done(), r.clusterRolebindingInformer.Informer().HasSynced); !ok {
```

Comment:
I'd rather having r.clusterRolebindingInformer.Informer() set during NewReconciler which then gives you a hassynced func.

Reply:
As @zhujian7 suggested, I use the lister interface in newReconciler, I think it's better.